src/helpers/daily-data-types/availability-check.ts

```diff
@@ -12,4 +12,27 @@ export function simpleAvailabilityCheck(namespace: DeviceDataNamespace, type: st
 			return false;
 		});
 	}
+}
+
+// TODO temporary for now.
```

Comment:
Please ping when this is resolved

Reply:
took this mostly from this pending PR https://github.com/CareEvolution/MyDataHelpsUI/pull/429, so thought is this will be replaced when that PR goes in

Reply:
If 429 is still WIP, can some testing be added for simpleAvailabilityCheckV2 and the TODO comment cleaned out?